python/tvm/testing/utils.py

```diff
@@ -84,6 +84,8 @@ def test_something():
 from tvm.relay.op.contrib.ethosn import ethosn_available
 from tvm.relay.op.contrib import cmsisnn
 
+SKIP_SLOW_TESTS = os.getenv("SKIP_SLOW_TESTS", "").lower() in {"true", "1", "yes"}
```

Comment:
what's the rationale for having 3 ways to do it?

Reply:
same as for the slow test triggers below, unlike skip ci where that was almost intentionally hard to use, this is something "user-facing" so it should accept all reasonable input (i.e. remembering that it has to be `1` instead of `yes` when tools like cmake accept either is confusing for no reason). Same for triggering slow tests, if someone's intent is clear but they don't remember the exact magic words we should still run the tests.

Reply:
okay the cmake precedent makes sense to me